lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/IMRUDriver.cs

```diff
@@ -243,6 +279,45 @@ public void OnNext(IFailedEvaluator value)
         }
 
         /// <summary>
+        /// This is to remove failed context from _activeContexts
+        /// More details will be implemented when working on REEF-1251
+        /// </summary>
+        /// <param name="value"></param>
+        private void RemovedFailedContext(IFailedEvaluator value)
```

Comment:
"first port the current behavior to the new structure " means refactor, correct? This is what the PR doing without adding fault tolerant logic yet. But to do it, we need to add some basic data structure to support it. 
It is hard to add some classes without usages, because we usually find issues when using it. 
Another way is to work on the fault tolerant directly because all the changes are about IMRUDriver. They are all related, it is hard to split. I can split few failed handlers while I am moving on. What do you think @markusweimer ?